daemon/create.go

```diff
@@ -244,8 +245,22 @@ func (daemon *Daemon) mergeAndVerifyConfig(config *containertypes.Config, img *i
 }
 
 // Checks if the client set configurations for more than one network while creating a container
+// Also checks if the IPAMConfig is valid
 func (daemon *Daemon) verifyNetworkingConfig(nwConfig *networktypes.NetworkingConfig) error {
-	if nwConfig == nil || len(nwConfig.EndpointsConfig) <= 1 {
+	if nwConfig == nil || len(nwConfig.EndpointsConfig) == 0 {
+		return nil
+	}
+	if len(nwConfig.EndpointsConfig) == 1 {
```

Comment:
If `> 1` then it will be an error condition (more than one endpointsConfig) which falls below to Ln 266 - 271.

On the other hand since `nwConfig.EndpointsConfig` is a map so I am using `for` to get the value of the map (though it can only have one key-value).